indico/MaKaC/registration.py

```diff
@@ -585,6 +585,13 @@ def getETicket(self):
             self._eTicket = eticket.ETicket()
             return self._eTicket
 
+    def hasStatusDefined(self, status_name):
+        for status in self._statuses.values():
```

Comment:
return any(status.getCaption() == status_name for status in self._statuses.itervalues())